selendroid-server/src/main/java/io/selendroid/server/model/AndroidNativeElement.java

```diff
@@ -478,6 +479,12 @@ public Point getLocation() {
     return new Point(xy[0], xy[1]);
   }
 
+  @Override
+  public Rectangle getRect() {
+    Point location = getLocation();
```

Comment:
Can you take a look at the `click` implementation? Turns out that `getLocation` might not always work immediately :/

I think we can do some refactoring to make it safer